addons/mail/static/src/models/message_seen_indicator_view.js

```diff
@@ -0,0 +1,38 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { one } from '@mail/model/model_field';
+import { clear, insertAndReplace, replace } from '@mail/model/model_field_command';
+
+registerModel({
+    name: 'MessageSeenIndicatorView',
+    identifyingFields: ['messageViewOwner'],
+    recordMethods: {
+        /**
+         * @private
+         * @returns {FieldCommand}
+         */
+        _computeMessageSeenIndicator() {
+            if (!this.messageViewOwner.threadView) {
+                return clear();
+            }
+            if (!this.messageViewOwner.threadView.thread) {
+                return clear();
+            }
+            return insertAndReplace({
+                message: replace(this.messageViewOwner.message),
+                thread: replace(this.messageViewOwner.threadView.thread),
+            });
+        },
```

Comment:
I think this is cleaner: 
```suggestion
        _computeMessageSeenIndicator() {
            if (this.messageViewOwner.threadView && this.messageViewOwner.threadView.thread) {
                return insertAndReplace({
                    message: replace(this.messageViewOwner.message),
                    thread: replace(this.messageViewOwner.threadView.thread),
                });
            }
            return clear();
        },
```